sqlx_todo/.env-example

```diff
@@ -0,0 +1,4 @@
+HOST=127.0.0.1
+PORT=5000
+DATABASE_URL="postgres://user:pass@192.168.33.11/actix_sqlx_todo"
+RUST_LOG=actix_rest_api_sqlx=info,actix=info
```

Comment:
actix_rest_api_sqlx doesnt match the package name. is this intended?

Reply:
Hi @robjtede, no it is not intended, it is because I was building it on separate project with this name "actix_rest_api_sqlx" and then mostly just copy it here and make readme... I will fix this now, sorry

Reply:
I have pushed update. Thanks for help